com.getpostman.Postman.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.getpostman.Postman</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>Postman</name>
+  <summary>Postman is a complete API development environment.</summary>
+  <description>
+    <p>
+      Postman helps you be more efficient while working with APIs. Using
+      Postman, you can construct complex HTTP requests quickly, organize them
+      in collections and share them with your co-workers.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.getpostman.com/img/v2/postman/Postman-most-complete-toolchain.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://www.getpostman.com</url>
+  <releases>
+    <release version="6.1.3" date="2018-05-25"/>
```

Comment:
I would put `<releases>` at the end of the file (easy to find, maybe you want to maintain a list).

Reply:
I usually don't.